### ISSUE TITLE
/alts enhancements

### ISSUE THREAD

**Issue by Owehttamy** [WarzoneMC/Warzone#589]
**Describe the feature you'd like**
1. Whenever a staff member uses "/alts", the list of players would be color coded by active punishments.
Example: If an account on the list has an active ban, then their name would display a red color,
if an account on the list has an active mute, their name would be yellow. And maybe just for some consideration add some other color or difference showing that the account is playing on Warzone (not completely necessary but possibly beneficial)
On top of the color coordination, when hovering over the username of someone who's banned or muted, it shows the information of their mute/ban just like when hovering over the mutes and bans in "/p" or "/punishments"

2. Whenever a new player joins the server, staff can be sent information regarding if they have alts or not and if they're banned/muted, but won't send anything unnecessary to staff when there's no alts found.

**Usefulness of the feature**
Would make it much easier to check if a player is ban or mute evading so they can get punished earlier and more efficiently.

**Comment by AtditC** [comment]
What do you mean with "And maybe just for some consideration add some other color or difference showing that the account is playing on Warzone"? Do you mean it as in whether he is currently online?

**Comment by Owehttamy** [comment]
Yes, exactly. I reworded it now, so it should be easier to understand.

**Comment by Draem** [comment]
pr for this was merged

**Comment by BennyDoesTheStuff** [comment]
The second suggestion hasn't been completed yet

**Comment by Draem** [comment]
oh my mistake
